### ISSUE TITLE
Custom host?

### ISSUE THREAD

**Issue by ghveem** [Chardonneaur/ChatGPT#7]
I would love to be able to use a custom host like *.[openai.azure.com/](https://*.openai.azure.com/). Could that be possible?

**Comment by ronan-hello** [comment]
yes, actually there is a host field in the general settings that allow you to override host.
